### PR TITLE
Add scaffolding for using ci.centos.org and an initial job

### PR DESCRIPTION
**Pull request by ehelms**
The initial job being added is a PR test for Forklift containers.

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/centos.org/ansible/deprovision.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+- hosts: localhost
+  vars:
+    cico_data: "{{ lookup('file', './cico_data.json') | from_json }}"
+    api_key: "{{ lookup('file', '~/duffy.key') }}"
+  tasks:
+    - name: 'Destroy node'
+      cico:
+        action: done
+        api_key: "{{ api_key }}"
+        ssid: cico_ data.ssid
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/centos.org/ansible/deprovision.yml]: Do I need :eyeglasses: or is there a space between `_` and `data`?

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/centos.org/ansible/deprovision.yml]: Good find, odd that this did not error

**Comment by ekohl** [comment]
Could you look at the build failure?

**Comment by ekohl** [comment]
Will this close #481?

**Comment by ehelms** [comment]
This is now passing and working as intended in conjunction with https://github.com/theforeman/forklift/pull/685

**Pull request merged**